src/hotspot/share/opto/vectornode.hpp

```diff
@@ -153,6 +167,12 @@ class AddVBNode : public VectorNode {
   virtual int Opcode() const;
 };
 
+class SaturatingAddVBNode : public SaturatingVectorNode {
+ public:
+  SaturatingAddVBNode(Node* in1, Node* in2, const TypeVect* vt, bool is_unsigned) : SaturatingVectorNode(in1,in2,vt,is_unsigned) {}
```

Comment:
Style: spaces after the commas in `SaturatingVectorNode(in1,in2,vt,is_unsigned)`